### PR TITLE
DRILL-6834: Added tests to verify Drill option exec.return_result_set…

### PR DESCRIPTION
**Pull request by agozhiy**
…_for_ddl

### CODE REVIEW

File: framework/resources/Functional/no_result_set/DRILL_6834_set_option_no_result_set.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+set `exec.return_result_set_for_ddl` = false;
+--@test
+set `store.format` = 'json';
```
Comment (Agirish) [inline framework/resources/Functional/no_result_set/DRILL_6834_set_option_no_result_set.sql]: Not sure about what this is testing. May be the CTAS is missing?

File: framework/resources/Functional/no_result_set/DRILL_6834_use_no_result_set.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+set `exec.return_result_set_for_ddl` = false;
+use dfs.`drillTestDir`;
```
Comment (Agirish) [inline framework/resources/Functional/no_result_set/DRILL_6834_use_no_result_set.sql]: Same here. Not sure about what this is testing. May be the CTAS is missing?

File: framework/resources/Functional/no_result_set/DRILL_6834_select_query.sql [hunk 3]
```diff
@@ -0,0 +1,3 @@
+set `exec.return_result_set_for_ddl` = false;
+select first_name, last_name from cp.`employee.json` order by employee_id limit 5;
```
Comment (Agirish) [inline framework/resources/Functional/no_result_set/DRILL_6834_select_query.sql]: This is just a valid usecase right? The option is meant for DDLs. But sure, good to test if this works for all SQL statements.

**Comment by Agirish** [comment]
Hey @agozhiy , some of the tests look incomplete - can you please take a look and comment?

File: framework/resources/Functional/no_result_set/DRILL_6834_set_option_no_result_set.sql (hunk 1 above)
Comment (agozhiy) [reply framework/resources/Functional/no_result_set/DRILL_6834_set_option_no_result_set.sql]: This is not a DDL statement, but it is also affected by the option tested.

File: framework/resources/Functional/no_result_set/DRILL_6834_use_no_result_set.sql (hunk 2 above)
Comment (agozhiy) [reply framework/resources/Functional/no_result_set/DRILL_6834_use_no_result_set.sql]: Please see my previous comment.

File: framework/resources/Functional/no_result_set/DRILL_6834_select_query.sql (hunk 3 above)
Comment (agozhiy) [reply framework/resources/Functional/no_result_set/DRILL_6834_select_query.sql]: Yes, you're right. We need to be sure that if the option is false, regular SQL queries will return result.

**Comment by agozhiy** [comment]
@Agirish, I addressed your comments, please review.

File: framework/resources/Functional/no_result_set/DRILL_6834_set_option_no_result_set.sql (hunk 1 above)
Comment (Agirish) [reply framework/resources/Functional/no_result_set/DRILL_6834_set_option_no_result_set.sql]: So the SQL statement being tested here is the `set option=value`. Got it thanks.

**Pull request merged**